resources/assets/js/components/ChartDisplay.vue

```diff
@@ -1,26 +1,85 @@
-<script>
-    import { Bar, Pie, PolarArea, Doughnut } from 'vue-chartjs';
+<template>
+    <div v-if="chartType === 0">
+        <bar-chart :label="label10" :chart-data="localChartDataUpper" :options="chartOptions"></bar-chart>
+    </div>
+    <div v-else-if="chartType === 1">
+        <pie-chart :label="label10" :chart-data="localChartDataUpper"></pie-chart>
+    </div>
+    <div v-else-if="chartType === 2">
+        <polar-chart :label="label10" :chart-data="localChartDataUpper"></polar-chart>
+    </div>
+    <div v-else>
+        <doughnut-chart :label="label10" :chart-data="localChartDataUpper"></doughnut-chart>
+    </div>
+
+</template>
 
+<script>
+    import BarChart from "./BarChart";
+    import PieChart from "./PieChart";
+    import PolarChart from "./PolarChart";
+    import DoughnutChart from "./DoughnutChart";
     export default {
-        props : {
-        //     chartType: '',
-            label: 'Label'
+      components: {
+        BarChart,
+        PieChart,
+        PolarChart,
+        DoughnutChart
+      },
+      props: {
+        label10: {
+          type: String,
+          default: 'This is a label I want to create'
+        },
+        chartType:
+            {
```

Comment:
Indentation. You might want to lint this using something like https://github.com/vuejs/eslint-plugin-vue

Reply:
Fixed